src/FilterButtonList.js

```diff
@@ -0,0 +1,139 @@
+import F from 'futil-js'
+import _ from 'lodash/fp'
+import React from 'react'
+import { observer } from 'mobx-react'
+import {
+  Dynamic,
+  Flex,
+  CheckButton as DefaultCheckButton,
+  Modal as DefaultModal,
+  Popover as DefaultPopover,
+} from './layout'
+import DefaultIcon from './DefaultIcon'
+import InjectTreeNode from './utils/injectTreeNode'
+import styles from './styles'
+
+export let DefaultMissingTypeComponent = InjectTreeNode(({ node = {} }) => (
```

Comment:
This is duplicated from other parts of the code base, so let's factor it so it's not copied and pasted